#Channels are organized in a row-major, top-down fashion
uint32 h #horizontal resolution
uint32 v #vertical resolution

# Fiel of Views in degrees
float32 h_fov #horizontal fov
float32 v_fov #vertical fov
